tests/test-email-creation.js

```diff
@@ -0,0 +1,35 @@
+const chai = require('chai');
+const rewire = require('rewire');
+const nock = require('nock');
+const testUsers = require('./users.json');
+const utils = require('./utils.js');
+
+const emailCreationScheduler = rewire('../schedulers/emailCreationScheduler');
+
+describe('emailCreationScheduler', () => {
+  it('getUnregisteredOVHUsers', async (done) => {
+    utils.cleanMocks();
+
+    const newMember = "membre.nouveau"
+    nock(/.*ovh.com/)
+    .get(/^.*email\/domain\/.*\/account/)
+    .reply(200, [newMember]);
+
+    const getUnregisteredOVHUsers = emailCreationScheduler.__get__('getUnregisteredOVHUsers');
+    const result = await getUnregisteredOVHUsers(testUsers);
+    /**
+     * [
+        {
+          id: 'membre.actif',
+          fullname: 'Membre Actif',
+          missions: [ [Object] ],
+        }
+      ]
+     */
+    console.log('mytest', result);
+
+    result.should.be.equal([newMember]);
+
+    done();
```

Comment:
```
Error: Timeout of 3000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/app/tests/test-email-creation.js)
```

Reply:
dans ce cas la tu peux retirer le `done`

Reply:
Donc si il y a un `await`, tu ne peux pas utiliser `done`, Pour utiliser `done` il faudrait faire : 
```
getUnregisteredOVHUsers(testUsers).then(result => {
result.should.be.equal([newMember]);
done()
})
```

Reply:
A noter que dans certains cas (dotc un qu'on avait eu précédemment sur les tests du cron createEmail tu peux avoir :
```
it('getUnregisteredOVHUsers', (done) => {
// ...
  getUnregisteredOVHUsers(testUsers).then(async result => {
    await unFonctionAAppeller()
    result.should.be.equal([newMember]);
    done()
  })
```

La tu as un `await` et un `async` mais il ne sont pas sur la fonction de test `it` directement, donc ça marche quand même avec `done`.